content/manuals/engine/release-notes/27.md

```diff
@@ -23,6 +23,80 @@ For more information about:
 - Deprecated and removed features, see [Deprecated Engine Features](../deprecated.md).
 - Changes to the Engine API, see [Engine API version history](/reference/api/engine/version-history.md).
 
+## 27.4
+
+Release notes for Docker Engine version 27.4 releases.
+
+### 27.4.0
+
+{{< release-date date="2024-12-XX" >}} ################## TODO #########################3
+
+
+For a full list of pull requests and changes in this release, refer to the relevant GitHub milestones:
+
+- [docker/cli, 27.4.0 milestone](https://github.com/docker/cli/issues?q=is%3Aclosed+milestone%3A27.4.0)
+- [moby/moby, 27.4.0 milestone](https://github.com/moby/moby/issues?q=is%3Aclosed+milestone%3A27.4.0)
+- Deprecated and removed features, see [Deprecated Features](https://github.com/docker/cli/blob/v27.4.0/docs/deprecated.md).
```

Comment:
oh for the docs, we probably want to link to the documentation instead of GitHub, but I see we have a link to that at the top (line 23), so I think we can remove it here 🤔